src/Options.hs

```diff
@@ -19,13 +19,13 @@ import           Paths_xsd_isogen    (version)
 import           Data.List
 
 
-data GenType = Parser | Generator | Both
+data GenType = Parser | Generator | ParserAndGenerator
```

Comment:
There is the exactly same type in [xml-isogen](https://github.com/typeable/xml-isogen/blob/7559025abcf27f94155a98e4653706a9b422e1ab/src/Data/THGen/XML/Internal.hs#L27). We can reuse it, but then `xsd-isogen` will depend on `xml-isogen` (it doesn't right now, and it's probably a good thing).

Reply:
Yep, there is no need in reusing it. They are different.